updates unordered writes config docs (#4131)
grafana/loki: docs/sources/configuration/_index.md

# CLI flag: -ingester.max-global-streams-per-user
[max_global_streams_per_user: <int> | default = 0]

> # Whether out of order writes are be accepted.

Reviewer: ```suggestion
# When true, out of order writes are accepted.
```
